lib/factory.ml

```diff
@@ -95,7 +95,7 @@ module Str = struct
     match ptype_kind with
     | Ptype_record labels -> from_record ~loc ~type_name ~labels
     | Ptype_variant constructors -> List.map (from_constructor ~loc ~type_name) constructors
-    | Ptype_abstract -> Raise.Factory.unhandled_type_kind ~loc "abstract"
+    | Ptype_abstract -> []
```

Comment:
Is it worth logging a warning or something (not sure if this is allowed inside a ppx) so that the user knows they used an abstract type, knows we ignored it and has a link to an issue they can follow for resolution?

Reply:
I'll have a look but I'm not sure there's a well supported way to do that.